doc/code/qml_grouping.rst

```diff
@@ -46,7 +46,10 @@ the groups of observables:
     :no-inheritance-diagram:
     :no-inherited-members:
 
-
 .. automodapi:: pennylane.grouping.graph_colouring
     :no-inheritance-diagram:
     :no-inherited-members:
+
+.. automodapi:: pennylane.grouping.pauli_group
```

Comment:
I just realised that this is probably not needed, since the new functions are imported directly into `qml.grouping`, so appear in the main summary table?

![image](https://user-images.githubusercontent.com/2959003/112430840-a85a9500-8d79-11eb-8ad7-84ef9e59737f.png)